src/Esfa.Vacancy.Register.Api/Web.config

```diff
@@ -97,7 +99,7 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" culture="neutral" publicKeyToken="30ad4fe6b2a6aeed" />
-        <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-9.0.0.0" newVersion="9.0.0.0" />
```

Comment:
🔙 ?

Reply:
Since we are using an older Nest package which depends on older Json package and has limited itself to version less than 10.